framework/tasks/utils.py

```diff
@@ -0,0 +1,4 @@
+from framework.tasks import app as celery_app
+
```

Comment:
Decoupling website code from tasks code. The Node model shouldn't need to know about the implementation of the tasks architecture.